locales/en/kafkaoverview-v2.json

```diff
@@ -5,15 +5,13 @@
   "contactSalesCardMainText": "Contact sales to get a pre-paid subscription or modify limits of your current subscription.",
   "contactSalesCardTitle": "Contact sales",
   "dataTransfer": "Data transfer",
+  "dataTransfer_price": "$0.09 / GB",
```

Comment:
@jgiardino the copy in the doc was using spaces around `/`. I kept the spaces. Is this correct?

Reply:
I didn't see any objections to the spaces in the content document. And there were a lot of [comments](https://docs.google.com/document/d/1Csg8gNZnog_PvNlSWaZIzww8KZ0TUYhlOVCuwf7Y3gc/edit?pli=1&disco=AAAAZPM15D0) from the BU about these, where the format of `x / y` was used in their comments. From that, I'd say the spaces are expected.